lib/lrmd/lrmd_client.c

```diff
@@ -1672,7 +1673,10 @@ lrmd_api_get_recurring_ops(lrmd_t *lrmd, const char *rsc_id, int timeout_ms,
 
             lrmd_op_info_t *op_info = calloc(1, sizeof(lrmd_op_info_t));
 
-            CRM_CHECK(op_info != NULL, break);
+            if (op_info == NULL) {
+                rc = -ENOMEM;
+                break;
```

Comment:
This is only going to break out of the inner for loop.  Perhaps this is meant to be a return instead?

Reply:
Good point, I think the idea was to keep trying later entries, but if we can't allocate a tiny amount of memory there's no point